bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3333,6 +3333,11 @@ void ReplicaImp::tryToMarkCheckpointStableForFastPath(const SeqNum &lastCheckpoi
   sendToAllOtherReplicas(checkpointMessage, true);
 }
 
+void ReplicaImp::setIsCollectingState(bool newState) {
```

Comment:
I think it would be better to move this one to header, and use inline specifier.